start_bs.py

```diff
@@ -141,31 +144,43 @@ class SampleXYZ(Device):
     mount_pos = MountPositions("XF:17IDB-ES:AMX", name="mount_pos")
     two_click_low = TwoClickLowMag("XF:17IDB-ES:AMX{Cam:6}", name="two_click_low")
     low_mag_cam_reset_signal = EpicsSignal('XF:17IDB-CT:AMX{IOC:CAM06}:SysReset')
+    top_cam_reset_signal = EpicsSignal('XF:17IDB-CT:AMX{IOC:CAM09}:SysReset')
     gonio = GoniometerStack("XF:17IDB-ES:AMX{Gon:1", name="gonio")
     loop_detector = LoopDetector(name="loop_detector")
     top_aligner_fast = TopAlignerFast(name="top_aligner_fast", gov_robot=gov_robot)
     top_aligner_slow = TopAlignerSlow(name="top_aligner_slow")
+    gov_mon_signal = EpicsSignal("XF:17ID:AMX{Karen}govmon", name="govmon")
+    gonio_mon_signal = EpicsSignal("XF:17ID:AMX{Karen}goniomon", name="goniomon")
+    pyz_homer = PYZHomer("", name="pyz_homer")
+    dewar = Dewar("XF:17IDB-ES:AMX", name="dewar")
+    home_pins = home_pins_plan(gov_mon_signal, gonio_mon_signal, pyz_homer, gonio)
+    robot_arm = RobotArm("XF:17IDB-ES:AMX", name="robot_arm")
     cs700 = CryoStream("XF:17ID1:CS700:", name="cs700", atol=0.1)
 
-elif beamline == "fmx":  
+elif beamline == "fmx":
     from mxbluesky.devices import (WorkPositions, TwoClickLowMag, LoopDetector, MountPositions, 
-                                   TopAlignerFast, TopAlignerSlow, GoniometerStack)
-    from mxbluesky.devices.generic import CryoStream
+                                   TopAlignerFast, TopAlignerSlow, GoniometerStack, Dewar, RobotArm)
+    from mxtools.vector_program import VectorProgram
+    from mxbluesky.devices.auto_recovery import PYZHomer
+    from mxbluesky.devices.cryostream import CryoStream
+    from mxbluesky.plans.auto_recovery import home_pins_plan
+    from mxtools.vector_program import VectorProgram
+    from mxtools.flyer import MXFlyer
+    from mxtools.raster_flyer import MXRasterFlyer
+    from embl_robot import EMBLRobot
+    import setenergy_lsdc
+
```

Comment:
Should we merge some of these with the above? Maybe one above both with shared imports and if beamline in ["fmx", "amx"].

Reply:
That's a good idea. I'll merge them in a future PR